test/lock-parser/fixtures/restriction-overwrite/out.json

```diff
@@ -0,0 +1,232 @@
+{
+  "dependencies": [
+    {
+      "name": "FSharp.Core",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "4.0.0.1",
+      "remote": "https",
+      "repository": "NUGET",
+      "dependencies": []
+    },
+    {
+      "name": "FAKE",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "4.62.1",
+      "group": "Build",
+      "remote": "https",
+      "repository": "NUGET",
+      "dependencies": []
+    },
+    {
+      "name": "FSharp.Compiler.Service",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "2.0.0.6",
+      "group": "Build",
+      "remote": "https",
+      "repository": "NUGET",
+      "dependencies": []
+    },
+    {
+      "name": "FSharp.Formatting",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
+      },
+      "version": "2.14.4",
+      "group": "Build",
+      "remote": "https",
+      "repository": "NUGET",
```

Comment:
`registry` or `package-registry` would go better with what we already use internally in our code

Reply:
So this is what `paket` calls these. It's their specific terminology which we attempted to preserve so that there was alignment with `paket` ideologies for future devs needing to maintain/add settings here. We can change this.